website/docs/getting-started/build-a-sample-app.mdx

```diff
@@ -138,4 +139,4 @@ Trunk will rebuild your application if you modify any of its source code files.
 
 You have now successfully set up your Yew development environment, and built your first web application.
 
-Experiment with this application and review the [examples](./examples.mdx) to further your learning.
```

Comment:
I think the `examples.mdx` contains useful information  about the example.
So I am not in favour of people to skip this page.

Reply:
@futursolo I agree with you.
I didn't remove the:
`Experiment with this application and review the [examples](./examples.mdx) to further your learning.`
I replace by
`Experiment with this application and review the [examples](https://github.com/yewstack/yew/tree/yew-v0.20.0/examples) to further your learning.`